apartmenttherapy/order-service
# Elaine/change log

#### What does this PR do & why?
I had put in a `changeLog` column for orders in kind of a hand-wavy way of preparing the way for a proper event log feature, notes, etc. However it was only logging the order creation event, which is not very useful. This change just makes sure `changeLog` is getting populated with the full list of order change events.

Note this is not intended to be used in the actual log feature-- we'll need to implement a lot more framework/db structure around that. This is just meant to capture data in a digestible format until we have time to implement the proper feature. When the time comes we'll be able to migrate the data in `changeLog` to the structure we want, and everyone's order histories will magically populate. In theory 😁 

#### How can this be manually tested?
- Tests should run.
- Try running an order through a few status changes-- creation, acceptance, cancelation, etc. Verify the `changeLog` array in the db gets populated with what you'd expect.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/1114641008675773

#### GIF


#### Have you remembered to...?
